src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -1016,6 +1016,8 @@ int main(int argc, char** argv) {
   logger_init();
   logger_set_threshold(boost::log::trivial::trace);
 
+  // note: we used to run tests for all key types, which was pretty slow.
+  // Now, they only run with ed25519
   return RUN_ALL_TESTS();
```

Comment:
Did you mean for this comment to be here, or should it be in `repo_test.cc`? Otherwise, I'm in favor!

Reply:
Wow, what a mix-up... Thanks, I'll fix that